bundle/testcase.go

```diff
@@ -15,29 +15,37 @@
 package bundle
 
 import (
+	"encoding/json"
 	"io"
 
 	log "github.com/sirupsen/logrus"
 
 	"github.com/erda-project/erda/apistructs"
 	"github.com/erda-project/erda/bundle/apierrors"
+	"github.com/erda-project/erda/pkg/http/httpserver"
 	"github.com/erda-project/erda/pkg/http/httputil"
 )
 
-func (b *Bundle) CreateTestCase(req apistructs.TestCaseCreateRequest) ([]byte, error) {
+func (b *Bundle) CreateTestCase(req apistructs.TestCaseCreateRequest) (httpserver.Resp, error) {
```

Comment:
You should define a corresponding resp like `apistructs.TestCaseCreateResponse`.
Otherwise, the invoker will not know the response data structure.

Reply:
OK